providers/src/airflow/providers/amazon/aws/transfers/http_to_s3.py

```diff
@@ -164,12 +164,21 @@ def s3_hook(self) -> S3Hook:
     def execute(self, context: Context):
         self.log.info("Calling HTTP method")
         response = self.http_hook.run(self.endpoint, self.data, self.headers, self.extra_options)
-
-        self.s3_hook.load_bytes(
-            response.content,
-            self.s3_key,
-            self.s3_bucket,
-            self.replace,
-            self.encrypt,
-            self.acl_policy,
-        )
+        if self.extra_options.get("stream", False):
```

Comment:
Can you also please add documentation about this option. It's absolutely not discoverable now.

Reply:
Done